docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
+
+Most modules of the SAP Cloud SDK for JavaScript are Open Source released under Apache 2.0 license. It's preferred, but not mandatory that contribution happens under same conditions.
+
+## Roles
+
+    - **SDK team** - development team and Product Owner responsible for maintaining, developing and releasing SAP Cloud SDK for JavaScript.
+    - **Contribution team** - development team responsible for developing and contributing a feature or an API library
+
+### Stakeholders and contact points
+
+    - **SDK Team** assigns two main stakeholders as contact points for the contribution process.
+      - **Prodact owner**  for organisational topics, priorities and practicalities
+      - **Developer** for engineering onboarding, alignment and consulting
+    - **Contribution team** assigns at least one contact point with engineering background for regular collaboration and syncs.
+
+Other stakeholders and/or contact points could be assigned if deemed necessary.
+
+## Process outline
+
+Below are generic steps for a successful contribution. The process can be adopted for the needs of specific contribution.
+
+    - Initial alignment and approvals
+    - Plan conribution collaboration
+    - Prepare and configure required assests (repositories, configuration, piplines, etc)
+    - Onboarding session with SDK team. Get to `Hello World!`
+    - Start of development by contribution team
+    - SDK Team provides ongoing consulting
+    - Regular syncs on development and organisational topics
+    - Successful PoC
+    - Iterative development process with SDK team contributing to PR reviews
+    - Releasing the contribution as `Beta`
+    - Iterative development. Stabilizing the API.
+    - Release the contribution as GA aka `General Availability`
+    - Keep functional scope up to date with customer requirements
+    - Maintanance and support of the contribution by **Contribution Team** or their successors
+
+## Responsibilities
+
+### SDK Team
+
+#### On-boarding
+
+  - Providing repository structure
+  - ESlint rules (so that compliant with SDK)
+  - Inherit our pipeline (Github actions)
+  - Testing framework
+  - NPM audit (vulnerability checks)
+  - License check tool
+
+
+### What's on them?
+
+#### Delivery plan
+  - What features
+  - Oder of feature delivery (so that we can anticipate PR sizes... else?)
+  - Basic architecture - some diagram and involved services
+  - Modules structure (CC, re-use module artifacts)
+
+
+- Setting up OS process?
+  - Coding and testing the feature?
+  - Aligning on release strategy?
+
+## Coding convention
```

Comment:
There is also a sub paragraph ### Coding Convention. Perhaps distinguish the two as this beeing the general header.

Reply:
Also Link this to Phase II. The development process